src/RouteCollector.php

```diff
@@ -48,6 +45,16 @@ class RouteCollector
      */
     private $routes = [];
 
+    /**
+     * @var array
```

Comment:
Shouldn't it be `@var Route[]`?

Reply:
Yep, fixed and added description for second search structure.